File: source/OptimalTravelPlan/OptimalTravelPlanRWS/src/java/ws/DataModel/PaymentTransactionHandler.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package ws.DataModel;

import entity.Customer;
import entity.PaymentTransaction;

/**
 *
 * @author sucram
 */
public class PaymentTransactionHandler {
    private PaymentTransaction paymentTransaction;
    private Customer customer;
    private Long bookingId;
    private Long paymentAccountId;
    private String password;

    public PaymentTransaction getPaymentTransaction() {
        return paymentTransaction;
    }

    public void setPaymentTransaction(PaymentTransaction paymentTransaction) {
        this.paymentTransaction = paymentTransaction;
    }

    public Customer getCustomer() {
        return customer;
    }

    public void setCustomer(Customer customer) {
        this.customer = customer;
    }

    public Long getBookingId() {
        return bookingId;
    }

    public void setBookingId(Long bookingId) {
        this.bookingId = bookingId;
    }

    public Long getPaymentAccountId() {
        return paymentAccountId;
    }

    public void setPaymentAccountId(Long paymentAccountId) {
        this.paymentAccountId = paymentAccountId;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
    
}

File: source/DemoIonic/src/app/models/payment-transaction-handler.ts
import { PaymentTransaction } from "./payment-transaction";
import { Customer } from "./customer";

export class PaymentTransactionHandler {
    paymentTransaction: PaymentTransaction | undefined;
    customer: Customer | undefined;
    bookingId: number | undefined | null;
    paymentAccountId: number | undefined | null;
    password: string | undefined;

    constructor(paymentTransaction?: PaymentTransaction, customer?: Customer,
        bookingId?: number, paymentAccountId?: number,
        password?: string) {
            this.paymentTransaction = paymentTransaction;
            this.customer = customer;
            this.bookingId = bookingId;
            this.paymentAccountId = paymentAccountId;
            this.password = password;
    }


}
